theme/n21/views/authentication/home.hbs

```diff
@@ -28,7 +28,7 @@
                                     style="postion: absolute; top: 40%; font-size: 25px; position: absolute;"></i>
                                 <div class="placeholder" style="display: none;">
                                     Beim Laden des Inhalts der Startseite ist ein Fehler aufgetreten.
-                                    <button type="button" class="btn btn-primary mt-1">Erneut Versuchen</button>
+                                    <button type="button" class="btn btn-primary mt-1">Erneut versuchen</button>
```

Comment:
why is internationalization label is not used here?

Reply:
all this n21/home.hbs is not using internationalization, so I would leave it as it is

Reply:
because in this file all text are used in that way, I did not want to mix two approaches there

Reply:
Looks for me, that the usage of translations wasn't complete 
Maybe this is a follow up?

Reply:
for that we should create another ticket and close it this one